src/components/Message/components/Button/Button.js

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+const CENTER = 'center';
+const TOP = 'top';
+
+const baseStyles = ({ theme }) => css`
+  label: message__button;
+  display: block;
+  padding-left: ${theme.spacings.giga};
+  margin-left: auto;
+  flex-grow: 0;
+  flex-shrink: 0;
+`;
+
+const alignmentStyles = ({ align }) => {
+  const alignments = {
+    [CENTER]: 'center',
+    [TOP]: 'flex-start'
+  };
+  return css`
+    label: message__button--${align};
+    align-self: ${alignments[align]};
+  `;
+};
+
+/**
+ * Button used in the Message component. Used for styling and aligment
+ * purposes only.
+ */
+const MessageButton = styled('div')(baseStyles, alignmentStyles);
```

Comment:
Thinking about the action (not close) button here:

Hm, what does the design spec say about the style of these buttons? If I remember the screenshots correctly, the buttons didn't look at all like our buttons.

This would be bad. If we need more button styles, those should be part of the button spec.

In any case, if the messages require specific button styles, should `Message` maybe export the Specific button as a component?

Reply:
The action button is different from the spec that we currently have. I spoke Liam and he said that he wants to add more button styles. 

@melmdesign, maybe you wanna explain yourself? :)